guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -13,18 +13,28 @@ You only need the MAC address for the BMC interface if the BMC interface is mana
 ====
 
 .Procedure
-. Enable BMC on the {SmartProxy} server if it is not already enabled:
-.. Configure BMC power management on {SmartProxyServer} by running the `{foreman-installer}` script with the following options:
+. Install `ipmitool` on your {SmartProxy}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-bmc=true \
---foreman-proxy-bmc-default-provider=ipmitool
+# {project-package-install} ipmitool
 ----
-.. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
-.. From the list in the *Actions* column, click *Refresh*.
-The list in the *Features* column should now include BMC.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Enable BMC power management on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-bmc-default-provider=ipmitool \
+--foreman-proxy-bmc=true
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. From the list in the *Actions* column, click *Refresh*.
+Ensure that the list of features includes *BMC*.
+. Navigate to *Infrastructure* > *Subnets*.
+. Select the subnet of your host.
+. On the *Proxies* tab, select your {SmartProxy} as *BMC Proxy*.
```

Comment:
```suggestion
. On the *{SmartProxies}* tab, select your {SmartProxy} as *BMC {SmartProxy}*.
```
Unfortunately, these are branded. I need to double check.

Reply:
I've checked & applied branding. I think I can live with this too.

Reply:
Is this rather a Satellite vs. Non-Satellite thing?

Reply:
Yes, fixed.